guides/framework-guides/framework-svelte.mdx

```diff
@@ -207,27 +207,27 @@ For example, `airbnb-review.svelte`:
 Next, create a Novu workflow with an email step. This code-first notification workflow approach makes it easy for product teams to modify notification content.
 
 ```jsx
-// src/lib/echo/index.ts
+// src/lib/workflows/index.ts
```

Comment:
Similar to the `create-novu-app`, let's make sure we're putting these under `src/lib/novu/workflows.ts`